website/docs/docs/mesh/iceberg/snowflake-iceberg-support.md

```diff
@@ -8,7 +8,13 @@ description: Understand Snowflake support for Apache Iceberg.
 dbt supports materializing the table in Iceberg table format in two different ways:
 
 - The model configuration field `table_format = 'iceberg'` (legacy)
-- Catalog integration in the model, resource, or dbt_project.yml configuration
+- Catalog integration can be configured in the SQL config (inside the `.sql` model file), property file (model folder), or project the file &mdash; [`dbt_project.yml`](/reference/dbt_project.yml)
+
+:::tip **Catalog integration configuration**
+
+You can also create a `catalogs.yml` at the [top level of your dbt project](/docs/mesh/iceberg/snowflake-iceberg-support#configure-catalog-integration-for-managed-iceberg-tables).
```

Comment:
Can we expound upon this? It's a little vague and leaves me wondering how it's a tip